src/Chapter2.hs

```diff
@@ -620,7 +640,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+-- takeEven [] = []
+takeEven (x0:_:xs) = (x0 : takeEven xs)
+takeEven lst = lst
```

Comment:
Since this function is rather small, I would recommend adding patterns for the remaining cases explicitly. Just to make sure that GHC pattern checker does it work properly, as well as for self-checking
```suggestion
takeEven [] = []
takeEven [x] = [x]
```